From: Ubuntu Core Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 14 Feb 2019 18:59:14 -0800
Subject: gdbm-import

# DP: suggest installation of python3-gdbm package on failing _gdbm import
# DP: suggest installation of python3-gdbm package on failing _gdbm import
---
 Lib/dbm/gnu.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Lib/dbm/gnu.py b/Lib/dbm/gnu.py
index b07a1de..24ba36f 100644
--- a/Lib/dbm/gnu.py
+++ b/Lib/dbm/gnu.py
@@ -1,3 +1,6 @@
 """Provide the _gdbm module as a dbm submodule."""
 
-from _gdbm import *
+try:
+    from _gdbm import *
+except ImportError as msg:
+    raise ImportError(str(msg) + ', please install the python3-gdbm package')
